runtime/src/utxo.rs

```diff
@@ -39,10 +34,11 @@ pub struct Transaction {
 #[derive(PartialEq, Eq, PartialOrd, Ord, Default, Clone, Encode, Decode, Hash, Debug)]
 pub struct TransactionInput {
     /// Reference to an UTXO to be spent
-    pub parent_output: H256,
+    pub outpoint: H256,
```

Comment:
Is this a typo? Should it be `output`?

Reply:
I changed it to outpoint, to be consistent with utxo naming conventions. yeah sounds weird but that's what its called